lib/lvm/attributes.rb

```diff
@@ -2,7 +2,7 @@
 
 module LVM
   module Attributes
-    VERSION = "0.2.8"
+    VERSION = "0.2.8".freeze
```

Comment:
shouldn't this match a value with `VERSION` file?

Reply:
The both were at the time of this PR. The VERSION file is the source of truth for expeditor and is used by build the changelog version information. That info gets updated here via the update script. Both are 0.3.0 now